specification/migrate/common-types/resource-management/v1/types.json

```diff
@@ -0,0 +1,578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "Resource": {
+      "title": "Resource",
```

Comment:
Why are these being copied over from specification/common-types/resource-management into the "migrate" folder? Whats the intent for doing this and why cant the common types be directly referenced?

Reply:
Same question for all common types

Reply:
If this folder isn't here , seeing exception in breaking change validations , will check what's the reason for it?

Reply:
removed the folder